source/mainnet/net/mobile-wallet/share-address-mw.rst

```diff
@@ -6,25 +7,39 @@ Find and share your account address
 
 If you need to share your account address with someone, you can find the address on the account page.
 
-Read the following to learn how to share your account address.
+.. tabs::
 
-Share your account address
-==========================
+   .. tab:: |bw|
```

Comment:
Should we have this for the browser wallet here, in the mobile wallet section?

Reply:
Yes, because I can put the same topic in the TOC to cover all.

Reply:
However, I have now moved it to the Use Browser Wallet topic instead. I may change  that later though back to how it was.